src/data/zcl_abapgit_data_utils.clas.abap

```diff
@@ -71,7 +72,7 @@ CLASS zcl_abapgit_data_utils IMPLEMENTATION.
         CALL METHOD lo_obj->('IF_XCO_DBT_FIELDS~GET_NAMES')
           RECEIVING
             rt_names = rt_names.
-      CATCH cx_sy_dyn_call_illegal_class.
+      CATCH cx_sy_dyn_call_illegal_class cx_xco_news_exception.
```

Comment:
`cx_xco_news_exception` is not released, perhaps use `cx_xco_runtime_exception` instead?

https://abapedia.org/steampunk-2302-api/

Reply:
and `cx_xco_runtime_exception` probably does not exists in 702, so it cannot be used either

and `cx_xco_runtime_exception` is a `cx_no_check`

what kind of error does the current code create in the system?

Reply:
When using the feature "Beta - Data" to add a table to my repository, the mentioned exception will be thrown on my machine and the table wont be added.

Reply:
the abapGit code must be 702 compatible

`cx_xco_runtime_exception` does not even exist in 750